.travis.yml

```diff
@@ -49,11 +49,6 @@ env:
 
 before_install:
   - |
-    # General settings
-    set -e
-    stty cols 120
-    ulimit -c unlimited -S || true
```

Comment:
Could you return back 
```
ulimit -c unlimited -S || true
```

and
```
set -e
```

This first one is needed to generate core dumps, and `set -e`  to exit immediately if a command exits with a non-zero status.